setup.cfg

```diff
@@ -42,7 +42,7 @@ setup_requires =
     setuptools_scm
 install_requires =
     amplitf
-    expertsystem == 0.6.9
+    expertsystem @ git+https://github.com/ComPWA/expertsystem@sympy
```

Comment:
We have to remember to change this back and update before the merge

Reply:
Requires a (pre-)release of https://github.com/ComPWA/expertsystem/pull/454 first

Reply:
Fixed by 454980a